[2023-03-03T07:44:52.836+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: metadata_scraper.fetch_nexrad_stations manual__2023-03-03T07:44:47.596700+00:00 [queued]>
[2023-03-03T07:44:52.860+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: metadata_scraper.fetch_nexrad_stations manual__2023-03-03T07:44:47.596700+00:00 [queued]>
[2023-03-03T07:44:52.861+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-03-03T07:44:52.862+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2023-03-03T07:44:52.864+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-03-03T07:44:52.891+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): fetch_nexrad_stations> on 2023-03-03 07:44:47.596700+00:00
[2023-03-03T07:44:52.900+0000] {standard_task_runner.py:55} INFO - Started process 122 to run task
[2023-03-03T07:44:52.921+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'metadata_scraper', 'fetch_nexrad_stations', 'manual__2023-03-03T07:44:47.596700+00:00', '--job-id', '53', '--raw', '--subdir', 'DAGS_FOLDER/scrapper.py', '--cfg-path', '/tmp/tmpxp_z9khl']
[2023-03-03T07:44:52.923+0000] {standard_task_runner.py:83} INFO - Job 53: Subtask fetch_nexrad_stations
[2023-03-03T07:44:53.062+0000] {task_command.py:388} INFO - Running <TaskInstance: metadata_scraper.fetch_nexrad_stations manual__2023-03-03T07:44:47.596700+00:00 [running]> on host 951056ee0f21
[2023-03-03T07:44:53.163+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Charu Singh
AIRFLOW_CTX_DAG_ID=metadata_scraper
AIRFLOW_CTX_TASK_ID=fetch_nexrad_stations
AIRFLOW_CTX_EXECUTION_DATE=2023-03-03T07:44:47.596700+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-03T07:44:47.596700+00:00
[2023-03-03T07:44:53.485+0000] {taskinstance.py:1768} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 192, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/scrapper.py", line 170, in nexrad_stations
    df = pd.read_fwf("/opt/nexrad-stations.txt", colspecs=cols, skiprows=[1])
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/parsers/readers.py", line 762, in read_fwf
    return _read(filepath_or_buffer, kwds)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/parsers/readers.py", line 482, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/parsers/readers.py", line 811, in __init__
    self._engine = self._make_engine(self.engine)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/parsers/readers.py", line 1040, in _make_engine
    return mapping[engine](self.f, **self.options)  # type: ignore[call-arg]
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/parsers/python_parser.py", line 1203, in __init__
    PythonParser.__init__(self, f, **kwds)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/parsers/python_parser.py", line 96, in __init__
    self._open_handles(f, kwds)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/parsers/base_parser.py", line 229, in _open_handles
    errors=kwds.get("encoding_errors", "strict"),
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/common.py", line 707, in get_handle
    newline="",
FileNotFoundError: [Errno 2] No such file or directory: '/opt/nexrad-stations.txt'
[2023-03-03T07:44:53.519+0000] {taskinstance.py:1323} INFO - Marking task as FAILED. dag_id=metadata_scraper, task_id=fetch_nexrad_stations, execution_date=20230303T074447, start_date=20230303T074452, end_date=20230303T074453
[2023-03-03T07:44:53.547+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 53 for task fetch_nexrad_stations ([Errno 2] No such file or directory: '/opt/nexrad-stations.txt'; 122)
[2023-03-03T07:44:53.589+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2023-03-03T07:44:53.656+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
